pkg/chunk/cached_store.go

```diff
@@ -433,7 +433,7 @@ func (s *wSlice) upload(indx int) {
 		}
 		if s.store.conf.Writeback {
 			stagingPath, err := s.store.bcache.stage(key, block.Data, s.store.shouldCache(blen))
-			if err != nil {
+			if err != nil && !errors.Is(err, errStageConcurrency) {
```

Comment:
this is not correct now when err is errStageConcurrency

Reply:
Right. Sorry just had too many context switches this afternoon and lost track of the code.